client-id/handler_test.go

```diff
@@ -205,3 +205,20 @@ func Test(t *testing.T) {
 		})
 	}
 }
+
+func Test_Options(t *testing.T) {
+	options := []client_id.Option{
+		client_id.WithStore(store.NewLocal()),
+		client_id.WithHeaderExtractor("X-Client-ID"),
+		client_id.WithRequired(),
+	}
+
+	middleware := client_id.New(options...)
+
+	request := httptest.NewRequest(http.MethodOptions, "/", nil)
+
+	response := doRequest(request, middleware)
+	defer response.Body.Close()
+
+	require.NotEqual(t, http.StatusUnauthorized, response.StatusCode)
```

Comment:
Is this the only status that we don't want it to be?

Reply:
Sure we can test that it is 200